### PR TITLE
Don't fail when terminal says 0 columns

### PR DESCRIPTION
**Pull request by libnoon**
Some terminals say their width is 0.  In that case, use some default
width rather than displaying ugly warnings.

See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=942759

### CODE REVIEW

**Comment by liske** [comment]
Thanks for the PR, I would like to merge it but:

- I would not like to switch to `Term::ProgressBar` since it adds additional dependencies (and `Termin::ProgressBar` is not that common)
- please split the documentation comments into another PR (which I would like to merge)

**Comment by libnoon** [comment]
Done!
The documentation PR is https://github.com/liske/needrestart/pull/169.

File: perl/lib/NeedRestart/UI.pm [hunk 1]
```diff
@@ -39,6 +39,23 @@ sub new {
     }, $class;
 }
 
+# my $columns = &_get_columns();
+#
+# Return the number of columns to use for output.
+sub _get_columns {
+    my $default_columns = 80;           # Sane default
+    if (-t *STDOUT) {
+        my ($columns) = GetTerminalSize(\*STDOUT);
```
Comment (liske) [inline perl/lib/NeedRestart/UI.pm]: Please pass the filehandle twice due to Debian Bug#824564 (as already done in wprint).

**Comment by libnoon** [comment]
Thomas Liske writes:

> +        my ($columns) = GetTerminalSize(\*STDOUT);

> Please pass the filehandle twice due to Debian Bug#824564 (as already done in wprint).

Done!

--
Fabrice BAUZAC-STEHLY
PGP 015AE9B25DCB0511D200A75DE5674DEA514C891D

**Comment by liske** [comment]
Thanks!

**Pull request merged**